bundle/build.rs

```diff
@@ -0,0 +1,130 @@
+use cid::multihash::{Code, MultihashDigest};
+use cid::Cid;
+use fil_actor_bundler::Bundler;
+use std::error::Error;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::process::{Command, Stdio};
+use std::thread;
+
+/// Cargo package for an actor.
+type Package = str;
+
+/// Technical identifier for the actor in legacy CodeCIDs and else.
+type ID = str;
+
+const ACTORS: &[(&'static Package, &'static ID)] = &[
+    ("system", "system"),
+    ("init", "init"),
+    ("cron", "cron"),
+    ("account", "account"),
+    ("multisig", "multisig"),
+    ("power", "storagepower"),
+    ("miner", "storageminer"),
+    ("market", "storagemarket"),
+    ("paych", "paymentchannel"),
+    ("reward", "reward"),
+    ("verifreg", "verifiedregistry"),
+];
+
+const IPLD_RAW: u64 = 0x55;
+const FORCED_CID_PREFIX: &str = "fil/6/";
+
+fn main() -> Result<(), Box<dyn Error>> {
+    // Cargo executable location.
+    let cargo = std::env::var_os("CARGO").expect("no CARGO env var");
+    println!("cargo:warning=cargo: {:?}", &cargo);
+
+    let out_dir = std::env::var_os("OUT_DIR").expect("no OUT_DIR env var");
+    println!("cargo:warning=out_dir: {:?}", &out_dir);
+
+    // Compute the package names.
+    let packages = ACTORS
+        .iter()
+        .map(|(pkg, _)| String::from("fvm_actor_") + pkg)
+        .collect::<Vec<String>>();
+
+    let manifest_path = {
+        let curr_dir = std::env::current_dir()?;
+        let workspace_dir = curr_dir.parent().unwrap();
+        workspace_dir.join("Cargo.toml")
+    };
```

Comment:
This works even when used as a dependency of another project?

Reply:
Yes, it does. Because:

> In addition to environment variables, the build script’s current directory is the source directory of the build script’s package.

https://doc.rust-lang.org/cargo/reference/build-scripts.html#inputs-to-the-build-script